download_and_create_reference_datasets/v03/write_clinvar_ht.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import tempfile
+
+import hail as hl
+
+from hail_scripts.reference_data.constants import GCS_PREFIXES
+from hail_scripts.utils.clinvar import (
+    download_and_import_latest_clinvar_vcf,
+    parsed_clnsig,
+    parsed_clnsigconf,
+    CLINVAR_ASSERTIONS_LOOKUP,
+    CLINVAR_DEFAULT_PATHOGENICITY,
+    CLINVAR_PATHOGENICITIES_LOOKUP,
+    CLINVAR_GOLD_STARS_LOOKUP,
+)
+from hail_scripts.utils.hail_utils import write_ht
+
+CLINVAR_HT_PATH = 'clinvar/clinvar.GRCh{genome_version}.{timestamp}.ht'
+PARTITIONS = 100 # per https://github.com/broadinstitute/seqr-loading-pipelines/pull/383
+
+def run(environment: str):
+    for genome_version in ['37', '38']:
+        with tempfile.NamedTemporaryFile(suffix='.vcf.gz') as tmp_file:
+            mt = download_and_import_latest_clinvar_vcf(genome_version, tmp_file)
+            timestamp = hl.eval(mt.version)
```

Comment:
just want to double check that this global will also be present on the final ht

Reply:
should be there on this line: https://github.com/broadinstitute/seqr-loading-pipelines/pull/430/files/515135c47a496173421f57e277caf2ac4a8d7bc2#diff-a49ce7f83dbd25ead01293ef8d1eed2f0c0e79aac5607c74511d916be6848b8fR125

Reply:
I just wanted to make sure that none of the selects we are doing overwrite the global, but I realize that you have to explicitly `select_globals` to do that